### ISSUE TITLE
Thinking of adding DragGAN?

### ISSUE THREAD

**Issue by gptbert** [IDEA-Research/Grounded-Segment-Anything#278]
https://github.com/Zeqiang-Lai/DragGAN

**Comment by rentainhe** [comment]
> https://github.com/Zeqiang-Lai/DragGAN

Yes, we're very willing to add DragGAN demo in Grounded-SAM, but we did not find the best way to combine DragGAN with other algorithms, we will highlight the project first to let others see this great re-implementation~
